variables.tf

```diff
@@ -1,25 +1,36 @@
 variable "awp_cloud_account_id" {
-    description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
-    type        = string
+  description = "CLOUDGUARD_ACCOUNT_ID or EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+}
+
+variable "awp_hub_external_account_id" {
+  description = "centralized account EXTERNAL_AWS_ACCOUNT_NUMBER"
+  type        = string
+  default     = null
+}
+
+variable "awp_organization_id" {
```

Comment:
Why do we need the client to give us the org id?
We can take it from Mongo

Reply:
done